### ISSUE TITLE
Workflow fails

### ISSUE THREAD

**Issue by gregdavill** [gregdavill/setup-riscv-gnu-toolchain#2]
Sometimes I've seen this workflow fail. It might be related to hitting an API limit on github, but maybe some more debugging is needed.

Default API limit is 50? per hour. But with a token you get 1000. So that seems like a solution.
I've tried to add in the github workflow token to the api requests but this doesn't seem to help, or it's not correctly being passed through environment variables into curl.

**Comment by dlmiles** [comment]
https://github.com/gregdavill/setup-riscv-gnu-toolchain/blob/main/dist/install-riscv-gnu-toolchain#L34 is this curl use here applying the GITHUB_TOKEN header while downloading the main targz ?

**Comment by gregdavill** [comment]
You're right! And it also appears I had issues with my bash variable expansion so the curl arguments weren't correctly being passed through.

This might be fixed now, but I'll try it on a few repos first.
